### ISSUE TITLE
Install numpy and cython before requirements.txt

### ISSUE THREAD

**Issue by jpgallegoar** [yerfor/MimicTalk#5]
Pip install numpy<1.24 and pip install cython before installing requirements 

Submitted PR

**Comment by steven850** [comment]
@jpgallegoar Did you get mimictalk running on windows?

**Comment by jpgallegoar** [comment]
No, had many issues trying to install mmcv and finally gave up

**Comment by nitinmukesh** [comment]
@jpgallegoar 

mmcv is easy to install. I actually installed easily but facing another problem during inference
https://github.com/yerfor/MimicTalk/issues/10

**Comment by steven850** [comment]
> No, had many issues trying to install mmcv and finally gave up

Yeah same here, the one I was never able to get past, after fixing the other 10+ issues was this one here. 

copying mmcv\ops\csrc\pytorch\npu\voxelization_npu.cpp -> build\lib.win-amd64-3.10\mmcv\ops\csrc\pytorch\npu
      running build_ext
      error: [WinError 2] The system cannot find the file specified
      [end of output]
      
shame, I really wanted to try this out.
